feder/cases/models.py

```diff
@@ -130,6 +146,9 @@ class Case(TimeStampedModel):
     response_received = models.BooleanField(
         verbose_name=_("Response received"), default=False
     )
+    is_quarantied = models.BooleanField(
```

Comment:
I think it should be "is_quarantined" - https://translate.google.pl/?sl=en&tl=pl&text=quarantined&op=translate